free-tunes/ViewModels/ExploreViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ExploreViewModel.swift
+//  free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/02/25.
+//
+
+import Foundation
+
+protocol ExploreViewModelDelegate: AnyObject{
```

Comment:
Maybe you can add this when you are ready to add the functionality for it 😄

Reply:
makes sense, I'll edit it as suggested👍